src/table/cells/handle-key-press.js

```diff
@@ -1,5 +1,25 @@
 import { selectCell } from '../selections-utils';
 
+const handleChangePage = (newPage, setPageInfo, rowsPerPage, setPage) => {
+  setPageInfo({ top: newPage * rowsPerPage, height: rowsPerPage });
+  setPage(newPage);
+};
+
+export const updatePage = (event, totalRowSize, page, rowsPerPage, setPageInfo, setPage) => {
+  const pageSize = Math.floor(totalRowSize / rowsPerPage);
+  const leftRowSize = totalRowSize % rowsPerPage;
+  const isRightKeys = event.shiftKey && event.ctrlKey && event.key === 'ArrowRight';
```

Comment:
As most applications, ctrl on windows corresponds to cmd on mac, which is `metaKey` on the event. So need to check that is either ctrl or meta